presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/mvp/view/adapters/ImagesAdapter.java

```diff
@@ -30,14 +33,21 @@ public ImagesAdapter(List<Image> images) {
     @NonNull
     @Override
     public ImagesViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
-        View itemView = LayoutInflater.from(parent.getContext()).inflate(R.layout.card,parent,false);
-        return new ImagesViewHolder(itemView);
+        Context context = parent.getContext();
+
+        if(context != null){
+            View itemView = LayoutInflater.from(context).inflate(R.layout.card,parent,false);
+            return new ImagesViewHolder(itemView);
+        }
+        return null;
     }
 
     @Override
     public void onBindViewHolder(@NonNull ImagesViewHolder holder, int position) {
         holder.cardImageId.setText(String.format("%s %d", holder.itemView.getContext().getResources().getString(R.string.image_id), images.get(position).getId()));
 
+        holder.imageId = images.get(position).getId();
```

Comment:
You are accesing two times to "images.get(position).getId()". Create a local variable to do this

Reply:
done